### DISCUSSION TITLE
Error with member "strings"

### DISCUSSION THREAD

**Discussion by daniele-baccega** [FLAMEGPU/FLAMEGPU2#1135] [Q&A] [answered]
Hi,
I'm configuring FLAME GPU 2 on a new machine (with a NVIDIA GeForce RTX 4060, with Fedora as OS) and, after installing all the requirements, I've obtained the error in the image at compilation time. Maybe I installed a library in the wrong way. Can you help me? Thanks!

![Screenshot from 2023-10-27 17-50-15](https://github.com/FLAMEGPU/FLAMEGPU2/assets/94167431/93c9c2c1-21ed-4a07-97e7-36294a6d3fcc)

**Comment by ptheywood** [comment, marked as answer]
You've discovered a bug in the guarding of the strings object when `FLAMEGPU_SEATBELTS` is set to `OFF` - this appears to be a gap in our CI coverage. It looks like this was introduced in #1116

Until we fix this, you can either:

* Reconfigure with `-DFLAMEGPU_SEATBELTS=ON` - this will result in slower simulations (but with more helpful error messages)
* Use an older version of FLAMEGPU2 from before this bug was introduced, by changing `FLAMEGPU_VERSION` to point to a commit which worked, and enabling `FLAMEGPU_VERSION_ALLOW_HASH` if pointing to a specific hash rather than the most recent release tag. 1269658c4525798b98ad472aa150e73572febdfe is the most recent commit which does not contain this error. 

    ```
    set(FLAMEGPU_VERSION "1269658c4525798b98ad472aa150e73572febdfe" CACHE STRING "FLAMEGPU/FLAMEGPU2 git branch or tag to use")
    set(FLAMEGPU_VERSION_ALLOW_HASH "ON")
    ```

This is unlikely to be fixed in `master` until next week at the earliest.

**Reply by Robadob** [reply]
Sorry @daniele-baccega, my fault. PR https://github.com/FLAMEGPU/FLAMEGPU2/pull/1139 has the fix.

Once CI has passed I'll probably push it straight to `master`. It's a trivial fix, so I'm not bothered about a review.

**Reply by Robadob** [reply]
This is now fixed in `master`.
